docs/basics.txt

```diff
@@ -1,10 +1,8 @@
 [[basics]]
-JanusGraph Basics
-=================
+= JanusGraph Basics
 
 [[configuration]]
-Configuration
--------------
+== Configuration
 
 A JanusGraph graph database cluster consists of one or multiple JanusGraph instances. To open a JanusGraph instance a configuration has to be provided which specifies how JanusGraph should be set up.
```

Comment:
_"JanusGraph server cluster consists of one or more JanusGraph server instances."_ sounds good to me. Done.